src/Controller/EventsController.php

```diff
@@ -467,9 +467,10 @@ public function putAction(Request $request, PDO $db): void
         if (!isset($request->url_elements[4])) {
             // Edit an Event
             $event_mapper   = new EventMapper($db, $request);
-            $existing_event = $event_mapper->getEventById($event_id, true);
 
-            if (!$existing_event) {
+            try {
+                $event_mapper->getEventById($event_id, true);
```

Comment:
I think we can drop the try-catch-rethrow here since the event mapper returns a sane(r) exception for this.